incuna-sass/_normalize.sass

```diff
@@ -44,7 +44,7 @@ html
 // Links
 
 // Remove the gray background color from active links in IE 10.
-@if support-legacy-browser("ie", "10")
+.lt-ie11 &
```

Comment:
maybe we take this whole `a background: transparent` out but make a note in the issue to check this and check if it needs replacement or something